api/src/db/migrations/v65_add_nexus_location_to_profile.ts

```diff
@@ -0,0 +1,292 @@
+import {v65UserData} from "./v65_add_task_progess_card";
+
+export interface v66UserData {
+  user: v66BusinessUser;
+  profileData: v66ProfileData;
+  formProgress: v66FormProgress;
+  taskProgress: Record<string, v66TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v66LicenseData | undefined;
+  preferences: v66Preferences;
+  taxFilingData: v66TaxFilingData;
+  formationData: v66FormationData;
+  version: number;
+}
+
+export const migrate_v65_to_v66 = (v65Data: v65UserData): v66UserData => {
+  return {
+    ...v65Data,
+    profileData: {
+      ...v65Data.profileData,
+      nexusLocationInNewJersey: undefined
+    },
+    version: 66,
+  };
```

Comment:
Do you need to add a field for the dba name?

Reply:
yep! wip :)